pom.xml

```diff
@@ -156,14 +159,15 @@
                                     <extensionArgs>-XautoNameResolution</extensionArgs>
                                 </xsdOption>
                             </xsdOptions>
+                            <sourceRoot>${project.build.directory}/generated-sources</sourceRoot>
                         </configuration>
                     </execution>
                 </executions>
             </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.22.2</version>
+                <version>3.0.0-M5</version>
```

Comment:
can you explain why we want to upgrade to M5 specifically? is it needed to run any of the tests with `jakarta` deps?

if so, and we need to upgrade to `3.x`, why don't we upgrade to `3.2.5` which is the latest version instead of a `-M` version?

when I've tried locally it seems like things work with `2.22.2` as well as the others, but perhaps my setup is lying to me (it does sometimes)

Reply:
Oh sorry I just missed this comment, I have fixed the version to 3.2.5. It is not related to jakarta it was just upgrading the version. It is up to you if you need to have the upgraded or revert please let me know.